Software-Engineering.md

```diff
@@ -133,6 +133,24 @@ _We avoid distinguishing too much between the Junior Software Engineer and Softw
 - I can confidently and charismatically pitch ideas, positively influencing my team to take decisive action based on reaching consensus.
 - I am an adept communicator, and can effectively steer technical and non-technical conversations to positive outcomes over any medium.
 
+## Principal Software Engineer
+
+> I am an influential Engineering [leader](Leadership.md) who identifies problems and opportunities relevant to Octopus' strategy, builds consensus to invest in solutions, and guides Engineering to deliver significant lasting change across Octopus.
```

Comment:
Should we mention "Mission"s here, and give "mission"s a clearer definition? I think it would aid in clarity.

Reply:
Concensus was that adding terminology around "missions" here doesn't improve the position description, so we will opt not to.